test/compilable/isreturnonstack.d

```diff
@@ -0,0 +1,8 @@
+
+struct S { int[10] a; }
+int test1();
+S test2();
+
+static assert(__traits(isReturnOnStack, test1) == false);
+static assert(__traits(isReturnOnStack, test2) == true);
+
```

Comment:
I suggest adding a test where a struct is not returned on the stack.

Reply:
ok

Reply:
It's quite a few different conditions and branches in the `isReturnOnStack` function. Are those covered by other tests? I guess they're implicitly covered by other tests?

Reply:
Check the green lines in the code tab, they are lines that are covered.

Reply:
Are you referring to this https://codecov.io/gh/dlang/dmd/compare/bde2b0aee87d3199db22d606a63fe034c48f1c61...87a0811b3fd408c54e77569f3f3c666e8c44203c/diff ? I see quite a few lines that are red instead of green.

Reply:
All the new code is green. The move of `retStyle` from one module to another is not new code. Also, the coverage tester does not test Windows specific code.